src/page-modules/assistant/trip/index.tsx

```diff
@@ -136,6 +136,7 @@ function TripPattern({ tripPattern, delay }: TripPatternProps) {
 
   const duration = secondsToDuration(tripPattern.duration, language);
   const fromPlace = tripPattern.legs[0]?.fromPlace.name ?? '';
+  const fromMode = tripPattern.legs[0]?.mode ?? 'unknown';
```

Comment:
So from the comment below this might be `tripPattern.legs.find(l => l.mode != 'foot')` or something. Check in the app how that works

Reply:
I see, will expand on this then!